R/pkg/R/column.R

```diff
@@ -44,6 +44,9 @@ setMethod("initialize", "Column", function(.Object, jc) {
   .Object
 })
 
+#' @rdname column
+#' @name column
+#' @aliases column,jobj-method
```

Comment:
ah, I see. that's kinda bad - it could be confusing to find this and it is not callable.